src/script/page/MainContent/panels/preferences/accountPreferences/AccountInput.tsx

```diff
@@ -182,20 +184,32 @@ const AccountInput: React.FC<AccountInputProps> = ({
           label={label}
           name={valueUie ? valueUie : fieldName}
           value={input}
+          inputWrapperRef={inputWrapperRef}
```

Comment:
This ref is never `set` (passed in a `ref={inputWrapperRef}` prop) it will always be `null` here